test/test_errhandler.py

```diff
@@ -8,6 +8,43 @@ def testPredefined(self):
         self.assertFalse(MPI.ERRHANDLER_NULL)
         self.assertTrue(MPI.ERRORS_ARE_FATAL)
         self.assertTrue(MPI.ERRORS_RETURN)
+        if MPI.VERSION >= 4:
+            self.assertTrue(MPI.ERRORS_ABORT)
+        else:
+            self.assertFalse(MPI.ERRORS_ABORT)
+
+    def testGetErrhandler(self):
+        errhdls = []
+        for i in range(100):
```

Comment:
Nothing in this loop depends on `i`.  What is it doing?

Reply:
Perhaps I should have written `for _ in range(100):`, as the loop variable is indeed not used . The purpose of the test is to stress a bit MPI's internal reference counting for error handlers.